packages/compass-e2e-tests/helpers/commands/index.ts

```diff
@@ -41,8 +41,7 @@ export * from './get-feature';
 export * from './set-feature';
 export * from './save-favorite';
 export * from './save-connection-string-as-favorite';
-export * from './select-connection';
-export * from './select-connection-menu-item';
+export * from './sidebar-connection';
```

Comment:
I combined a bunch of sidebar connection related commands into one file to mirror what we do for workspace management already. Otherwise it is a bit of a proliferation of little files. It might also help if we do move these to page objects because they'd already be together.